homeassistant/components/growatt/sensor.py

```diff
@@ -0,0 +1,178 @@
+"""
+Support for Growatt Plant energy production sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.growatt/
+"""
```

Comment:
We do not require link in docstring anymore

Reply:
Removed it.